superset/connectors/sqla/models.py

```diff
@@ -830,6 +831,17 @@ def init_on_load(self) -> None:
     def __repr__(self) -> str:
         return str(self.column_name)
 
+    @validates("python_date_format")
+    def validate_pdf_is_iso8601(self, _: str, dt_format: str) -> str:
```

Comment:
The function name is somewhat misleading as the Python date format doesn't strictly need to be ISO 8601 seeing as we support the `epoch_s` and `epoch_ms` values.